SingularityService/src/main/java/com/hubspot/singularity/auth/datastore/SingularityAuthDatastore.java

```diff
@@ -1,9 +1,10 @@
 package com.hubspot.singularity.auth.datastore;
 
-import com.google.common.base.Optional;
+import java.util.Optional;
+
 import com.hubspot.singularity.SingularityUser;
 
 public interface SingularityAuthDatastore {
   Optional<SingularityUser> getUser(String username);
-  Optional<Boolean> isHealthy();
+  com.google.common.base.Optional<Boolean> isHealthy();
```

Comment:
Do we want to mix Google and Java `Optional`s in the Singularity codebase?

Reply:
Singularity is still on guava optionals, new dropwizard uses java util. It would be a breaking change in the client to move everything over. We need to do it at some point. ButI'm not sure this PR is the place yet